src/oas2/transformers/params.ts

```diff
@@ -74,7 +75,7 @@ export function translateToHeaderParams(headers: { [headerName: string]: Header
 }
 
 export function translateToBodyParameter(body: BodyParameter, consumes: string[]): IHttpOperationRequestBody {
-  const examples = map(get(body, 'x-examples'), (value, key) => ({ key, value }));
+  const examples = map(get(body, 'x-examples') || getExamplesFromSchema(body.schema), (value, key) => ({ key, value }));
```

Comment:
It seems to me we're handling the `x-examples` twice; once here and once in the inner function? Is this confusing?

```suggestion
  const examples = map(getExamplesFromSchema(body.schema), (value, key) => ({ key, value }));
```

Reply:
First one refers to root level example (`body`), while second to schema example (`body.schema`) so those might be two different `x-examples`